projects/golang/go/build/conformanceTests/eksDistroRebuildBuildspec.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file was generated by running `make generate`. DO NOT EDIT
+
+version: 0.2
+
+batch:
+  fast-fail: true
+  build-graph:
+    - identifier: k8s_122
+      env:
+        variables:
+          RELEASE_BRANCH: 1-22
+    - identifier: k8s_123
+      env:
+        variables:
+          RELEASE_BRANCH: 1-23
+    - identifier: k8s_124
+      env:
+        variables:
+          RELEASE_BRANCH: 1-24
+    - identifier: k8s_125
+      env:
+        variables:
+          RELEASE_BRANCH: 1-25
+
+env:
+  variables:
+    ARTIFACTS_BUCKET: "eks-d-postsubmit-artifacts"
+    ARTIFACT_BUCKET: "eks-d-postsubmit-artifacts"
+    BUILDKIT_PLATFORMS: "linux/amd64"
+    CONTROL_PLANE_INSTANCE_PROFILE: "arn:aws:iam::125833916567:instance-profile/KopsControlPlaneBuildRole"
+    DOCKER_CONFIG: "/home/prow/go/src/github.com/aws/eks-distro/.docker"
+    EKS_D_BUILD_TOOLING_REMOTE_URL: "https://github.com/aws/eks-distro-build-tooling.git"    
+    EKS_D_REMOTE_URL: "https://github.com/aws/eks-distro.git"
+    IMAGE_REPO: "public.ecr.aws/h1r8a7l5"
+    KOPS_STATE_STORE: "s3://testbuildstack-125833916-kopsbuildstatestorebucke-d4esen60nfrk"
+    NODE_INSTANCE_PROFILE: "arn:aws:iam::125833916567:instance-profile/KopsNodesBuildRole"
+    REBUILD_ALL: "true"
+    SKIP_CHECKSUM_VALIDATION: "true"
+    TARGET_BRANCH: "codebuildTest"
+    TEST_ROLE_ARN: "arn:aws:iam::125833916567:role/TestBuildRole"
+
+phases:
+  pre_build:
+    commands:
+      - echo Running pre-build setup and initialization...
+      - scripts/setup_ecr_credential_helper.sh
+      - scripts/setup_qemu.sh
```

Comment:
Is there no arm option when running on codebuild?

Reply:
there is but youd have to separate the build out across two instances/jobs then "merge" back together which for this job is probably unnecessary, most of the image builds are pretty simple and do not even trigger emulation.  As an example, we do use arm instances for one of the builds in the eks-a [staging](https://github.com/aws/eks-anywhere-build-tooling/blob/main/release/staging-build.yml#L182) build.